test/tests.jl

```diff
@@ -1,462 +1,465 @@
+    @test convert(Rational,kappa) == 1//2
+
+    u = SymFunction("u")
+    eqn = Eq(x^2 + u(x)^2, x^3 - u(x))
+    @test diff(eqn, x) == Eq(2x + 2u(x) * diff(u(x),x), 3x^2 - diff(u(x),x))
+
+    ## integrate
+    @test integrate(sin(x)) == -cos(x)
+    @test integrate(sin(x), (x, 0, pi)) == 2.0
+    a, b, t = symbols("a, b, t")
+    @test integrate(sin(x), (x, a, b)) == cos(a) - cos(b)
+    @test integrate(sin(x), (x, a, b)) |> replace(a, 0) |> replace(b, pi) == 2.0
+    @test integrate(sin(x) * DiracDelta(x)) == sin(Sym(0))
+    @test integrate(Heaviside(x), (x, -1, 1)) == 1
+    C = Curve([exp(t)-1, exp(t)+1], (t, 0, log(Sym(2))))
+    @test line_integrate(x + y, C, [x,y]) == 3 * sqrt(Sym(2))
+
+
+    ## summation
+    summation(1/x^2, (x, 1, 10))
+    out = summation(1/x^2, (x, 1, 10))
+    out1 = sum([1//x^2 for  x in 1:10])
+    @test @compat round(Integer, out.x[:p]) == out1.num
+    @test @compat round(Integer, out.x[:q]) == out1.den
+
+
+    ## Ops
+    s = 3
+    x = Sym("x")
+    v = [x, 1]
+    rv = [x 1]
+    a = [x 1; 1 x]
+    b = [x 1 2; 1 2 x]
+
+    const DIMERROR = VERSION < v"0.4.0-dev" ? ErrorException : DimensionMismatch
+    const DimensionOrMethodError =  Union{MethodError, DimensionMismatch}
+    ## scalar, [vector, matrix]
+    @test s + v == [x+3, 4]
+    @test v + s == [x+3, 4]
+    @test s + rv == [x+3 4]
+    @test rv + s == [x+3 4]
+    @test s + a == [x+3 4; 4 x+3]
+    @test a + s == [x+3 4; 4 x+3]
+
+    @test s - v == [3-x, 2]
+    @test v - s == [x-3, -2]
+    @test s - rv == [3-x 2]
+    @test rv - s == [x-3 -2]
+    @test s - a == [3-x 2; 2 3-x]
+    @test a - s == [x-3 -2; -2 x-3]
+
+    2v
+    2rv
+    2a
+    s .* v
+    v .* s
+    s .* rv
+    rv .* s
+    s .* a
+    a .* s
+
+    ## s / v  ## broadcasts s Depreacated
+    @test s ./ v == [3/x, 3]
+    @test v / s == [x/3, Sym(1)/3]
+    @test v .\ s == s ./ v
+    @test s \ v == v / s
+    ## s / rv ## broadcasts s Deprecated
+    @test s ./ rv == [3/x 3]
+    @test rv / s == [x/3 Sym(1)/3]
+    @test rv .\ s == s ./ rv
+    @test s \ rv == rv / s
+    ## s / a ## broadcasts s Deprecated
+    @test s ./ a == [3/x 3; 3 3/x]
+    @test a / s == [x/3 Sym(1)/3; Sym(1)/3 x/3]
+    @test a .\ s == s ./ a
+    @test s \ a == a / s
+
+    @test_throws MethodError  s ^ v ## error
+    @test s .^ v == [3^x, 3]
+    @test_throws DimensionOrMethodError v ^ s ## error
+    v .^ s
+    @test_throws MethodError  s ^ rv ## error
+    s .^ rv
+    @test_throws DimensionMismatch  rv ^ s ## error
+    rv .^ s
+    @test_throws MethodError  s ^ a ## error
+    s .^ a
+    a ^ s
+    a .^ s
+
+
+    ## vector vector
+    @test v .+ v == 2v
+    ##@test_throws MethodError  v .+ rv ##  no longer an error, broadcase
+    @test v .- v == [0, 0]
+    @test_throws DIMERROR  v - rv
+    @test_throws DimensionOrMethodError   v * v ## error
+    @test v .* v == [x^2,1]
+    @test dot(v, v) == 1 + conj(x)*x
+    if VERSION >= v"0.6.0-dev"
```

Comment:
In this case, I think the newest v"0.6.0-dev" has reverted this behaviour, whereas for a while it was throwing an error. I think the best way would be to find the range between the two commits, but as this is testing against a moving development branch, maybe we (hopefully) don't need to be that fussy and can just drop the check on the VERSION for this test.